components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -21,7 +21,10 @@
 @interface MDCTabBarView ()
 
 /** The views representing the items of this tab bar. */
-@property(nonatomic, strong) NSArray<UIView *> *itemViews;
+@property(nonnull, nonatomic, strong) NSArray<UIView *> *itemViews;
+
+/** The title colors for bar items. */
+@property(nonnull, nonatomic, strong) NSMutableDictionary<NSNumber *, UIColor *> *titleColors;
```

Comment:
It was not immediately clear to me that this dictionary is mapping `UIControlState`s to `UIColor`s. Please update either the comment or the property name to reflect this. I'm not sure what our naming conventions are for dictionaries, but something like `statesToTitleColors` would be a lot clearer.

Reply:
Done.